gensrc/script/doris_builtins_functions.py

```diff
@@ -297,6 +297,20 @@
     [['array_intersect'], 'ARRAY_VARCHAR',   ['ARRAY_VARCHAR', 'ARRAY_VARCHAR'], '', '', '', 'vec', ''],
     [['array_intersect'], 'ARRAY_STRING',    ['ARRAY_STRING', 'ARRAY_STRING'], '', '', '', 'vec', ''],
 
+    # reverse function support the array type
```

Comment:
reverse for string and varchar type, also put here together with array type

Reply:
done